vim-patch:35e7594

justinmk opened the pull request:
This only converts the test since @watiko already merged the relevant old-style tests.

justinmk committed:
test: convert test_marks.in.

vim-patch:35e7594
https://github.com/vim/vim/commit/35e7594dd429f7a8a06cefd61c3e8d48b9bd74e2
    "Add missing test files from 7.4.634 to the repository."

The discrepancy between the expected getpos() result of the old test
[0, 15, 2, 0] and the converted test [0, 3, 2, 0] is just a matter of how
the buffer is constructed: in the old Vim test the buffer has a bunch of
junk at the top.

The central purpose of the test is to verify that the getpost("'a") does
*not* return [0, 0, 0, 0].

watiko commented:
I will update PR #4013 which include `vim-patch:7.4.634` and `vim-patch:7.4.1085`.

justinmk committed:
vim-patch.sh: recognize malformed patches.

vim-patch:fc39ecf (committed in aae7e8b)
vim-patch:541f92d (committed in cc52060)
vim-patch:88774fd (committed in 896f548)

This commit message forces vim-patch.sh to recognize these patches
which were committed with only 6 chars (vim-patch.sh requires 7).
